src/hypertrace/agent/config/test_unittests.py

```diff
@@ -0,0 +1,113 @@
+'''Unittest for merging config entries.'''
```

Comment:
I think a good convention would be `test_<entity_we_are_testing>`.py

Reply:
It was renamed to test_config.py.